pkg/db/migrations/migration_structs.go

```diff
@@ -25,7 +25,8 @@ import (
 //
 // 4. Create one function in a separate file that returns your Migration. Add that single function call to this list.
 var MigrationList = []*gormigrate.Migration{
-	addDinosaurs(),
```

Comment:
because we have no long running database instance anywhere, we can safely drop this migration. if we had a schema that depended on this migration in any way, it would have to be preserved so that all migrations always run cleanly.

Reply:
additionally, this PR as-is would break integration tests. Please mark it WIP.

Reply:
thanks for reminder, done.

Reply:
instead of changing the existing one, could we add an additional resource migration? And I can remove the old migration in PR #3 with updated test. So that we will not break the tests.

Reply:
yes, it is *always* preferable to add new migrations than remove old ones.  new migrations can drop tables.

Reply:
it makes sense!
dropped the `dinosaurs` and add `resources` & `consumers` in new migrations.